setup.cfg

```diff
@@ -21,13 +21,13 @@ packages =
 include_package_data = true
 
 install_requires =
+    billiard
```

Comment:
Fine thought we might want to replace that
```
>>> billiard.current_process().name
'MainProcess'
```
Seems not useful anymore.

Reply:
Can be replaced by dyno name from env for example